models/java/gbfs-java-model/pom.xml

```diff
@@ -195,7 +179,7 @@
                     <dependency>
                         <groupId>org.mobilitydata.gbfs</groupId>
                         <artifactId>rule-factory</artifactId>
-                        <version>1.0.0</version>
+                        <version>1.0.2</version>
```

Comment:
Should we have all the versions follow each other?

Reply:
I think it ads complexity to have people update all the files. We need to update gbfs-java-models for release purposes, but I don't think it's that important for the others